guides/common/modules/proc_altering-the-privilege-elevation-method.adoc

```diff
@@ -0,0 +1,22 @@
+[id="altering-the-privilege-elevation-method_{context}"]
+= Altering the Privilege Elevation Method
+
+By default, push-based remote execution uses `sudo` to switch from the SSH user to the effective user that executes the script on your host.
+In some situations, you may require to use another method, such as `su` or `dzdo`.
+You can globally configure an alternative method to switch to the effective user in your {Project} settings.
+
+.Prerequisite
+* If you want to use `dzdo` for Ansible jobs, you have to install the `community.general` Ansible collection, which contains the *dzdo* Ansible plug-in.
```

Comment:
```suggestion
* Ensure the Ansible collection that provides the required plug-in is installed.
For example, to use `dzdo`, ensure the `community.general` collection is installed.
```

Reply:
It's meant to be just for dzdo, because the other methods that Satellite provides (su, sudo), are part of ansible-core which is included by default and doesn't need an additional Ansible collection.

Reply:
I'd go with "module" instead of "plug-in"; unless you want to say "'become' plug-in": https://docs.ansible.com/ansible/latest/collections/community/general/index.html#plugin-index

Other than that, I like Anet's suggestions.

Reply:
@maximiliankolb @asteflova I've updated what I agreed with.